PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,289 @@
+
+**Tools and methods used as a reference**
+
+This methodology used the latest version of the following tools:
+- CDM methodological tool Demonstration of additionality of small-scale project activities
+- CDM methodological Tool for the demonstration and assessment of additionality
+
+This methodology is based upon approaches used in the following methodologies:
+
+- CDM methodology for Grid Connected electricity generation from renewable sources 
+- CDM Methodology for zero-emissions grid-connected electricity generation from renewable sources in Chile or in countries with merit order based dispatch
+- CDM Small-scale Methodology for Renewable Electricity generation for captive use and mini-grid.
+- [AMS-IA: Electricity generation by the user --- Version 19.0](https://cdm.unfccc.int/methodologies/DB/1TIFADHWTMIW25TAL778RLEFJ6AWBB)
+
+
+All of the relevant existing methodologies are published by the UN Clean Development Mechanism, which focuses on larger scale projects in emerging economies. The small-scale methodology for renewable sources can include projects such as tidal and hydro and can be as large as 15MW. These are still large installations when compared to the small arrays that are being placed individually on roof-tops of homes, offices, warehouses and commercial and industrial buildings.
+
+The methodology being presented by PeerCo is for even smaller-installations than outlined in the CDM methodology, between 30 kWp and up to 4.9 MWp. The micro-versus small-scale is a key distinction as the economics are quite different. In this way, the PeerCo methodology is new, but fits within the same structure, aims and similar emissions calculations to the CDM methodologies. 
+
+The other key difference is that the PeerCo methodology is intended to be used in any country where the delivery of micro-solar is not a requirement for building development or to meet an enforced carbon quota.
+
+**Additionality considerations**
+
+The concept of additionality is that a project or investment would not have happened were it not for the additional carbon revenue stream. However, there is sufficient market evidence to prove that without subsidy, small-scale solar, particularly urban solar, has a long repayment period resulting in a marginal business case. Any user of this methodology should use direct carbon funding to those organisations that are making these marginal investments. In countries that are not Kyoto Protocol Annex 1 Countries, there is an even higher threshold for meeting additionality. PeerCo is addressing this point by retaining a percentage of any income stream linked to the use of this methodology and co-investing these funds into community owned energy companies or energy efficiency measures in social housing. 
+
```

Comment:
That's an interesting strategy, can you say what percentage of income is re-invested and how you chose that number?  

Do you see a possibility of free-riders in your programs and is this investment meant to balance against those?

Reply:
We were initially addressing this for carbon market audiences and because we are now purely looking at EACs, the point made here is more for the carbon markets and we should potentially remove this. We don't see people 'free riding' but we want to - as much as possible - prove an ongoing journey of electrification and decarbonisation which means we would like to see them build on their investments. One solar provider did want to offer this kind of proof of reinvestment and another wanted to use the proceeds to invest in more metering. but this is likely to be on a case by case basis.